src/video_canister_package/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 120,
+  "trailingComma": "all",
+  "singleQuote": true
+}
```

Comment:
glaube das prettier file muss auf die äußerste Ebene vom Repo. ansonsten nutzt es das nicht für die Formatierung. Siehe z.b. die quotes in den files

Reply:
Bzw die Location passt eigentlich, wenn man den Ordner öffnet liegt es richtig. aber du hast nicht damit formatiert oder?

Reply:
resolved beea9bbe1dd16a17d4f185f418d4cae363e83867